app/controllers/admin/users_controller.rb

```diff
@@ -1,28 +1,21 @@
 module Admin
   class UsersController < Admin::ApplicationController
-    def confirm
-      user = User.find(params[:id])
-      confirmation_reminder = ConfirmationReminder.new(user)
+    # To customize the behavior of this controller,
```

Comment:
Don't remove the previous code, it has an extra action to resend confirmation email. We need that.